blocklist-client/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "blocklist-client"
+version = "0.0.1"
+edition = "2021"
+
+[[bin]]
+name = "blocklist-client"
+path = "src/main.rs"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+warp = "0.3"
+tokio = { version = "1", features = ["full"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+reqwest = { version = "0.11", features = ["json"] }
+tokio-serde = "0.8"
+env_logger = "0.9"
+log = "0.4"
```

Comment:
Did you mean `logging.rs` in signer crate? I made changes to use the logging that was defined there with the same tracing dependencies. Let me know if that's what we want.

Reply:
Right yeah, it's in the `signer` crate. Thank you for updating :pray:. We will likely need to break it out to a separate crate later to avoid cyclic dependencies as soon as the signer starts using the blocklist client, but we don't have to do that right now.